src/sentry/conf/server.py

```diff
@@ -70,8 +70,6 @@ def env(key, default="", type=None):
 
 ADMIN_ENABLED = DEBUG
 
-MAINTENANCE = False
```

Comment:
Could this have been used in open source? cc @BYK

Reply:
Yeah, I'm wondering if ops would ever set this...

Reply:
Nope, not in ~~open source~~ self-hosted @EvanPurkhiser